libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -113,23 +116,25 @@ void AP_CRSF_Telem::setup_custom_telemetry()
         return;
     }
 
-    // setup the frsky scheduler for crossfire
+    // setup the frsky scheduler for crossfire and elrs
     passthrough->disable_scheduler_entry(AP_Frsky_SPort_Passthrough::GPS_LAT);
     passthrough->disable_scheduler_entry(AP_Frsky_SPort_Passthrough::GPS_LON);
     passthrough->disable_scheduler_entry(AP_Frsky_SPort_Passthrough::TEXT);
     passthrough->set_scheduler_entry_min_period(AP_Frsky_SPort_Passthrough::ATTITUDE, 350); // 3Hz
 
     // setup the crossfire scheduler for custom telemetry
-    set_scheduler_entry(BATTERY, 1000, 1000);       // 1Hz
-    set_scheduler_entry(ATTITUDE, 1000, 1000);      // 1Hz
     set_scheduler_entry(FLIGHT_MODE, 1200, 2000);   // 0.5Hz
     set_scheduler_entry(HEARTBEAT, 2000, 5000);     // 0.2Hz
 
-    _telem_rf_mode = get_rf_mode();
+    _telem.rf_mode = get_rf_mode();
     // setup custom telemetry for current rf_mode
-    update_custom_telemetry_rates(_telem_rf_mode);
+    update_custom_telemetry_rates(_telem.rf_mode);
 
-    gcs().send_text(MAV_SEVERITY_DEBUG,"CRSF: custom telem init done, fw %d.%02d", _crsf_version.major, _crsf_version.minor);
+    if (_crsf_version.is_elrs) {
+        gcs().send_text(MAV_SEVERITY_DEBUG,"ELRS: custom telem init done, fw %d.%02d", _crsf_version.major, _crsf_version.minor);
```

Comment:
We can save a bit of flash by putting this reporting into a function

Reply:
ok